PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -22,6 +22,11 @@
         private let urlInterceptor: URLSessionInterceptor
         private var sessionSwizzler: URLSessionSwizzler?
 
+        // SwiftUI image types
+        private let swiftUIImageTypes = ["_TtCOCV7SwiftUI11DisplayList11ViewUpdater8Platform13CGDrawingView",
+                                         "_TtC7SwiftUIP33_A34643117F00277B93DEBAB70EC0697122_UIShapeHitTestingView",
+                                         "SwiftUI._UIGraphicsView", "SwiftUI.ImageLayer"].compactMap { NSClassFromString($0) }
```

Comment:
I suppose we won't know until we release but is there a chance that there are more auto-generated types in a customers app?

Reply:
Since the types are auto-generated, we cannot just do `view as? SwiftUI.Text` similarly to UIKit.
So yeah I believe this is a moving thing that we'll discover more over time or new ones will come up within the next few SwiftUI versions, but those are the "root" auto-generated types that work well with the basic widgets at least.

Reply:
Some of them are mapped to the UIKit though, for example:
TextField -> `UITextField`.
navigationTitle -> `UILabel`